src/app/cube/home/components/featured-collection-card/featured-collection-card.component.spec.ts

```diff
@@ -1,15 +1,16 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { FeaturedCollectionCardComponent } from './featured-collection-card.component';
 
 describe('FeaturedCollectionCardComponent', () => {
   let component: FeaturedCollectionCardComponent;
   let fixture: ComponentFixture<FeaturedCollectionCardComponent>;
 
-  beforeEach(async(() => {
+  beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
-      declarations: [ FeaturedCollectionCardComponent ]
-    })
+    declarations: [FeaturedCollectionCardComponent],
+    teardown: { destroyAfterEach: false }
```

Comment:
I know this teardown has been appearing a lot for the Angular testing, why is it needed now?

Reply:
This has always been a feature of Angular testing and was optional. As of Angular 13+, this teardown parameter is now required. The 'destroyAfterEach' is a flag for the unit tests to either destroy the TestBed after each individual test is ran or to reuse the same TestBed for all tests in that component. In our case, we want to reuse the same testbed for all tests associated with that component.